src/ui/DynamicFacet/DynamicFacetValues/DynamicFacetValues.ts

```diff
@@ -26,6 +27,10 @@ export class DynamicFacetValues implements IDynamicFacetValues {
     this.facetValues = response.values.map((facetValue, index) => this.valueCreator.createFromResponse(facetValue, index));
   }
 
+  public reorderValues(order: string[]) {
```

Comment:
It's not about reordering the values, it's about defining a set of currentValues before sending the request, kind of like setting a predefined range on the range facets

Reply:
Hmm, this wasn't requested originally.

Do you think it should just append the `customSort` values to the results or try to fetch them alongside other values on a search request to confirm they exist?

If it only gets appended, then some parameters such as `numberOfvalues` would be missing whenever the value wasn't part of the original search response.

Reply:
I am pretty sure the feature request can be rephrased as:

> Show me the facet value I am configuring at the position I am specifying, with the correct count

So the configured value would need to be sent in the request, along with any other params (e.g. freezeCurrentValues?) to ensure that value is returned in the response, even if the count is 0.

Also, we need to make sure that if I configure 2 values, but want to see 5, that the search-api still provides 3 other values.

Reply:
Update after scrum: we decided not to do this yet.